model/user.go

```diff
@@ -244,8 +244,8 @@ func (u *User) ToJson() string {
 }
 
 // Generate a valid strong etag so the browser can cache the results
-func (u *User) Etag() string {
-	return Etag(u.Id, u.UpdateAt)
```

Comment:
Hmmm that's not a bad idea, I think it might work but it would require some refactoring and we'd always have to make sure we're sanitizing the user object before generating the etag.

 I think I'd rather the small annoyance of forcing the passing in of two bools rather than having a constraint on the order of operations between sanitization and etag generation. Thoughts?